package.json

```diff
@@ -60,15 +59,16 @@
     "karma-requirejs": "1.1.0",
     "karma-sinon": "1.0.5",
     "karma-spec-reporter": "0.0.32",
-    "node-sass": "~3.7.0",
+    "mini-css-extract-plugin": "^1.0.0",
     "phantomjs-prebuilt": "2.1.16",
     "require-dir": "latest",
     "requirejs-plugins": "1.0.2",
     "run-sequence": "1.2.2",
-    "sass-loader": "3.2.3",
+    "sass": "^1.0.0",
+    "sass-loader": "^10.0.0",
     "style-loader": "0.23.1",
     "through2": "2.0.5",
-    "webpack": "2.7.0",
+    "webpack": "^4.0.0",
```

Comment:
Any reason why we're not upgrading to Webpack v5 as that is latest and the version our micro-frontends use across Open edX via frontend-build?